frontend/src/metabase/entities/metrics.js

```diff
@@ -26,8 +25,7 @@ const Metrics = createEntity({
 
   objectSelectors: {
     getName: metric => metric && metric.name,
-    getUrl: metric =>
-      Urls.tableRowsQuery(metric.database_id, metric.table_id, metric.id),
+    getUrl: metric => `/metric/${metric.id}`,
```

Comment:
Sneaking a fix for clicking metrics in search results.

----

Repro steps for the issue (without changes in this file):
1. Create a metric
2. Search for it via search in app header
3. Click the metric search result

App error is displayed

----

See [Slack conversation](https://metaboat.slack.com/archives/C051AG38B2S/p1713209487863139?thread_ts=1713196024.852279&cid=C051AG38B2S).